Source/Core/Core/HW/MemmapFunctions.cpp

```diff
@@ -853,53 +713,44 @@ static __forceinline u32 TranslatePageAddress(const u32 _Address, const XCheckTL
 
 	// hash function no 1 "xor" .360
 	u32 hash = (VSID ^ page_index);
+	u32 pte1 = bswap((VSID << 7) | api | PTE1_V);
```

Comment:
Poor neglected UPTE1 struct definition, nobody cares enough to make use of use you...